resources/views/components/address/address.blade.php

```diff
@@ -115,7 +115,7 @@
             {{ __('Date Of Birth') }}
         </label>
         <div class="col-span-2">
-            <x-datetime-picker wire:model="address.date_of_birth" :without-time="true" x-bind:disabled="!$wire.edit" />
```

Comment:
You can leave this as datetime picker, it's fine when you have to click on the year to type it